src/leaderboard/leaderboard.css

```diff
@@ -0,0 +1,98 @@
+* {
+  box-sizing: border-box;
+  background-color: #8ab0ab;
+  margin: 0;
+  padding: 0;
+  font-family: 'Bebas Neue', cursive;
+}
+
+#scorePage {
+  position: relative;
+  text-align: center;
+}
+
+#scorePageTitle {
+  font-size: 5rem;
+  font-weight: bold;
+  color: #2c3e50;
+}
+/*podium*/
+
+.podium {
+  position: relative;
+  display: flex;
+  justify-content: center;
+}
+
+.podiumNick {
+  margin-top: 30px;
+  margin-left: 15px;
+  display: flex;
+  justify-content: center;
+  text-transform: uppercase;
+  font-size: 2rem;
+  word-wrap: break-word;
+  max-width: 180px;
+  color: whitesmoke;
+}
+
+.step {
+  height: 3em;
+  padding: 5px;
+  vertical-align: bottom;
+}
+
+#podium3,
+#podium1,
+#podium2 {
+  display: flex;
+  width: 150px;
+  background: #2c3e50;
+  justify-content: center;
+  align-items: center;
+  font-size: 6rem;
+  margin: 20px 30px;
+  color: whitesmoke;
+}
+#podium1 {
+  /* gold */
+  height: 250px;
+}
+#podium2 {
+  /* silver */
+  height: 200px;
+}
+#podium3 {
+  /* bronze */
+  height: 150px;
+}
```

Comment:
try to avoid using names with numbers - it makes code unreadable. 
A better approach is to use nth-child() selectors